tests/vmi_multus_test.go

```diff
@@ -320,17 +320,6 @@ var _ = Describe("Multus", func() {
 					&expect.BExp{R: "1"},
 				}, 15)
 				Expect(err).ToNot(HaveOccurred())
-
-				By("Verifying the desired custom MAC is not configured inside the pod namespace.")
```

Comment:
Can you enter the pod and verify that it is removed from the bridge database instead of removing the tests?